### ISSUE TITLE
Error while serializing long values with VPackBuilder

### ISSUE THREAD

**Issue by christian-lechner** [arangodb/arangodb-java-driver#73]
Hello!

I think I've found a bug while serializing long values with [`VPackBuilder`](https://github.com/arangodb/arangodb-java-driver/blob/master/src/main/java/com/arangodb/velocypack/VPackBuilder.java):

```java
private static final Appender<Long> LONG = new Appender<Long>() {
	@Override
	public void append(final VPackBuilder builder, final Long value) throws VPackBuilderException {
		if (value <= 9 && value >= -6) {
			builder.appendSmallInt(value);
		} else {
			builder.add((byte) 0x23);
			builder.append(value, INTEGER_BYTES);
		}
	}
};
```
Shouldn't the line `builder.append(value, INTEGER_BYTES);` read `builder.append(value, LONG_BYTES);`?

The same for `BIG_INTEGER`?
```java
private static final Appender<BigInteger> BIG_INTEGER = new Appender<BigInteger>() {
	@Override
	public void append(final VPackBuilder builder, final BigInteger value) throws VPackBuilderException {
		if (value.longValue() <= 9 && value.longValue() >= -6) {
			builder.appendSmallInt(value.longValue());
		} else {
			builder.add((byte) 0x23);
			builder.append(value, INTEGER_BYTES);
		}
	}
};
```

Test code:
```java
long value = 12345678901L;
VPackBuilder builder = new VPackBuilder();
builder.add(value);
VPackSlice vpack = builder.slice();
System.out.println(vpack);   // output: 3755744309
```

**Comment by mpv1989** [comment]
Thanks,
I will fix it for version 4.1.3

Best
Mark

**Comment by christian-lechner** [comment]
Thanks Mark!

**Comment by mpv1989** [comment]
fixed in version 4.1.3
